payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -123,6 +123,10 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         return outputLiveData
     }
 
+    // Holds a reference to the last selected payment method while checking out with Google Pay.
```

Comment:
Is this needed to repopulate the values in the form?   Why do you need to reset it, what effect does it have?

Reply:
No, this is used because when you tap the Google Pay button on `PaymentSheetAddPaymentMethodFragment` it sets the payment method to Google pay, but when it fails or is cancelled we want to set back to the previously selected PM so the sheet is back into a usable state. Another option would be to get the PM from the fragment being shown, but that'll be more complicated because of the 2 types of fragment (for card, and compose).

Reply:
is this about setting the PM type (bancontact, sofort, ideal) in the selector, so the write form is present?